.mergify.yml

```diff
@@ -39,6 +39,25 @@ queue_rules:
       {{ title }} (#{{ number }})
       {{ body }}
 
+  - name: priority-merge
+    update_method: merge
+    merge_method: squash
+    conditions:
+      - base!=release
+      - -title~=(WIP|wip)
+      - -label~=(blocked|do-not-merge|no-squash)
```

Comment:
add a check to have `priority-pr` label .. like this `- label~=priority-pr`

Reply:
also update the `default-squash` queue conditions to check that the PR does not have `priority-pr` label .. update line 29